build.sbt

```diff
@@ -43,6 +43,10 @@ lazy val common_elasticsearch = doSharedLibrarySetup(project, "sbt_common/elasti
   .dependsOn(internal_model % "compile->compile;test->test")
   .settings(libraryDependencies ++= Dependencies.commonElasticsearchDependencies)
 
+lazy val common_messaging = doSharedLibrarySetup(project, "sbt_common/messaging")
+  .dependsOn(common % "compile->compile;test->test")
```

Comment:
Can we add a comment here to explain that this is because of the VHS dependency and to remove this once we take the VHS out?